### PR TITLE
Add grunt release and package metadata

### PR DESCRIPTION
**Pull request by chrisprice**
I've created the PR in case anyone wants to review it, but I'm going to merge this myself so that I can perform the release :)

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -257,8 +257,18 @@ module.exports = function (grunt) {
         },
 
         clean: {
+            dist: ['dist'],
```
Comment (jleft) [inline Gruntfile.js]: Won't this also delete the README in `dist`? Or is this the intention?

File: Gruntfile.js [hunk 2]
```diff
@@ -257,8 +257,18 @@ module.exports = function (grunt) {
...
             doc: ['doc'],
             visualTests: ['visual-tests/dist']
+        },
+
+        release: {
```
Comment (jleft) [inline Gruntfile.js]: Should this be a dev, or maybe optional, dependency (in package.json)?

**Comment by jleft** [comment]
Couple of comments, but very exciting to have an initial release! :smile:

File: Gruntfile.js (hunk 1 above)
Comment (chrisprice) [inline Gruntfile.js]: Good spot, fixed (9edb636812383abe7da90ab62bb02c237e484467)

File: Gruntfile.js (hunk 2 above)
Comment (chrisprice) [inline Gruntfile.js]: Fixed (31de299e80f4a5f1f062b60a89d82b53ce6dc4c7)

**Comment by chrisprice** [comment]
Thanks @jleft, I've fixed the issues you've pointed out but there's still problems with the process :( The build's not running before releasing, I'll take a look tomorrow.

**Pull request merged**